lib/WebDriverCheckbox.php

```diff
@@ -0,0 +1,72 @@
+<?php
```

Comment:
When I was using the classes I was thinking about their names - it seems they are still kinda confusing.

Consider this code:

```php
$checkbox = new WebDriverCheckbox($driver->findElement(...));

$checkbox->deselectAll();
$checkbox->getFirstSelectedOption();
```

It is confusing the class is called "checkbox" (ie. singular), however it actually represents not a single checkbox, but group of checkboxes - and also the methods semantically interacts with multiple objects (deselect*All*, get*First*Selected etc.).

So I have two possible suggestions how to make this more understandable:
1. Name it in plural, like "WebDriverCheckboxes", WebDriverRadios"
2. Or make it obvious it is group of elements: WebDriverCheckboxGroup", "WebDriverRadioGroup"

What do you think?